### PR TITLE
Update probes.mdx

### PR DESCRIPTION
**Pull request by edbshubham**
Added the Target Type elaboration for the probes.

### CODE REVIEW

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 1]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+    probes are used to collect data from different levels of the system hierarchy. The Target Type of a probe defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:169]:
```suggestion
     Probes are used to collect data from different levels of the system hierarchy. A probe's **Target Type** defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 2]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+    probes are used to collect data from different levels of the system hierarchy. The Target Type of a probe defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
+    gathers relevant metrics from the appropriate components of the system.
+
+    <details><summary>Target Type Configuration Matrix</summary>
+    | Target Type | Execution Level | Columns That the Probe Query Must return              | Probe examples |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:173]:
```suggestion
    | Target Type | Execution Level | Probe query columns            | Probe examples |
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 3]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+    |-------------|-----------------|-------------------------------------------------------|----------------|
+    | Agent       | Agent           | None                                                  | cpu_usage      |
+    | Server      | Server          | None                                                  |                |
+    | Database    | Database        | None                                                  |                |
+    | Schema      | Database        | schema_name                                           |                |
+    | Table       | Database        | schema_name, table_name                               |                |
+    | Index       | Database        | schema_name, index_name                               | index_size     |
+    | Sequence    | Database        | schema_name, sequence_name                            |                |
+    | View        | Database        | schema_name, view_name                                |                |
+    | Function    | Database        | schema_name,  arg_types, function_type, function_name |                |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:183]:
```suggestion
    | Function    | Database        | schema_name, arg_types, function_type, function_name |                |
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 4]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+    **Column Descriptions**
+    Target Type:
+    Specifies the target object that the probe will monitor. Each target type corresponds to a different kind of database object or level.
+
+    Execution Level:
+    Indicates the frequency at which the probe Executes Once Per object. This could be once per server, database, or other specified levels, ensuring that the monitoring is appropriately scoped.
+    Note that the Custom probe set to a target type of Database or higher (including Schema, Table, Index, View, Sequence, Functions) will be displayed at the database level and System level probes set to a target type of Schema or 
+    higher will be displayed until the schema level.
+
+    Columns That the Probe Query Must return:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:195]:
```suggestion
    Probe query columns:
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 5]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+    | Table       | Database        | schema_name, table_name                               |                |
+    | Index       | Database        | schema_name, index_name                               | index_size     |
+    | Sequence    | Database        | schema_name, sequence_name                            |                |
+    | View        | Database        | schema_name, view_name                                |                |
+    | Function    | Database        | schema_name,  arg_types, function_type, function_name |                |
+    | Extension   | Extension       | None                                                  | Extension      |
+
+    **Column Descriptions**
+    Target Type:
+    Specifies the target object that the probe will monitor. Each target type corresponds to a different kind of database object or level.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:188]:
```suggestion
    Specifies the target object that the probes monitor. Each target type corresponds to a different database object or level.
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 6]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+
+    Execution Level:
+    Indicates the frequency at which the probe Executes Once Per object. This could be once per server, database, or other specified levels, ensuring that the monitoring is appropriately scoped.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:191]:
```suggestion
    Indicates the level at which the probe executes. Probes are executed once per server, database, or other specified levels, ensuring appropriate monitoring scope.
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 7]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+    Note that the Custom probe set to a target type of Database or higher (including Schema, Table, Index, View, Sequence, Functions) will be displayed at the database level and System level probes set to a target type of Schema or
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:192]:
```suggestion
    Note that the custom probe set to a target type of Database or higher (including Schema, Table, Index, View, Sequence, and Functions) will be displayed at the database level and System-level probes set to a target type of Schema or 
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 8]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
...
+    Columns That the Probe Query Must return:
+    Lists the essential columns that must be included in the result set of the probe query. These columns ensure the necessary data is collected for the specified target type.
+
+    Examples:
+    Provides specific examples of probes for each target type, helping to illustrate how the probes are used in practice.
+
+    </details>
+    <br/>
+
+    Proper configuration of the Target Type in probes ensures that the collected metrics are relevant and accurate for the intended level of the system hierarchy. By setting the appropriate target type, users can effectively monitor
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:204]:
```suggestion
    Proper configuration of the **Target Type** in probes ensures that the collected metrics are relevant and accurate for the intended level of the system hierarchy. By setting the appropriate target type, users can effectively monitor
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx (hunk 2 above)
Comment (edbshubham) [reply product_docs/docs/pem/9/monitoring_performance/probes.mdx:173]: this column name should suggest that the probe query must have these columns these are not optional

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx (hunk 4 above)
Comment (edbshubham) [reply product_docs/docs/pem/9/monitoring_performance/probes.mdx:195]: same as above...need to come up with more appropriate name

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/probes.mdx:173]: How about "Mandatory columns" and rest we can describe in the column description?

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 9]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+    probes are used to collect data from different levels of the system hierarchy. The Target Type of a probe defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
+    gathers relevant metrics from the appropriate components of the system.
+
+    <details><summary>Target Type Configuration Matrix</summary>
```
Comment (djw-m) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:172]: This table below will not render. There's no blank line between the HTML and the Markdown table.

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx (hunk 2 above)
Comment (djw-m) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:173]:
```suggestion

        | Target Type | Execution Level | Required Probe Query Columns              | Probe examples |
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 10]
```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+     Probes are used to collect data from different levels of the system hierarchy. A probe's **Target Type** defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
+    gathers relevant metrics from the appropriate components of the system.
+
+    <details><summary>Target Type Configuration Matrix</summary>
+    | Target Type | Execution Level | Columns That the Probe Query Must return              | Probe examples |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:173]:
```suggestion
    
    | Target Type | Execution Level | Columns That the Probe Query Must return              | Probe examples |
```

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx (hunk 9 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/probes.mdx:172]: Review is in progress :-)

**Comment by nidhibhammar** [comment]
Closing this PR. The same changes are in progress here: #5914.

**Pull request closed without merging**